dev/integration_tests/flutter_gallery/lib/demo/material/grid_list_demo.dart

```diff
@@ -175,10 +73,16 @@ class GridDemoPhotoItem extends StatelessWidget {
           appBar: AppBar(
             title: Text(photo.title),
           ),
-          body: SizedBox.expand(
-            child: Hero(
-              tag: photo.tag,
-              child: GridPhotoViewer(photo: photo),
+          body: Hero(
+            tag: photo.tag,
+            child: InteractiveViewer(
```

Comment:
Should I even bother using InteractiveViewer in these two gallery demos anymore now that the main gallery is a separate repo?  I guess these are just for integration tests.